_posts/blog/2019-07-26-trusted-firmware-update-Jun19.md

```diff
@@ -0,0 +1,27 @@
+---
+author: trusted.firmware
+title: Trusted Firmware Project June 2019 Update 
+date: 2019-06-26 10:00:00
+categories:
+- Blog
+layout: post
+image: /assets/images/blog/musca_tf_crop_1500x1500.png
+---
+
+In this post there is a link to the progress update for last month. The linked presentation also includes updated roadmap and contribution statistics. 
+
+[Link to June 2019 TF update presentation](/docs/TrustedFirmware-Update-June-2019.pdf)
+
+For more information or to get involved, the Trusted Firmware Project maintains the following mailing lists for developers on mailman:
+```
+tf-a
+tf-m
+tf-a-tests
+```
+Details on how to join these lists are on the [Contacts page](https://www.trustedfirmware.org/contact/)
```

Comment:
Please do not use full URLs like this.

Instead please use this format:

`[Contacts page](/contact/)`

This advice only applies when the URL is within the same site, but you have used the correct format on line 13 above.